po/ja.po

```diff
@@ -14245,23 +14245,31 @@ msgid ""
 "very low and operating systems provide primitives for efficiently identifying\n"
 "I/O that is able to proceed."
 msgstr ""
+"「Async」は複数のタスクが並行処理される並行性モデルで、それぞれのタスクはブロックされるまで実行さ"
+"れ、そして次に進むことのできる他のタスクに切り替えることにより実現されます。このモデルは限られた数"
+"のスレッド上でより多くのタスクを実行することを可能にします。なぜなら、タスクごとのオーバーヘッドは"
+"通常はとても低く、効率的に実行可能なI/Oを特定するために必要なプリミティブをOSが提供してくれるからで"
+"す。"
 
 #: src/async.md:10
 msgid ""
 "Rust's asynchronous operation is based on \"futures\", which represent work that\n"
 "may be completed in the future. Futures are \"polled\" until they signal that\n"
 "they are complete."
 msgstr ""
+"Rustの非同期的な操作は「future」に基づいていて、これは将来に完了するかもしれない作業を体現していま"
+"す。Futureは、タスクが完了したことを知らせるシグナルが得られるまでポーリングされます。"
 
 #: src/async.md:14
 msgid ""
 "Futures are polled by an async runtime, and several different runtimes are\n"
 "available."
 msgstr ""
+"Futureは非同期的なランタイムによりポーリングされ、いくつかの異なるランタイムが利用可能です。"
```

Comment:
How about "Futureは非同期的なランタイムによりポーリングされます。ランタイムにはいくつかの選択肢があります。"? The first clause and the second clause in the sentence are for different topics. It would be less confusing if we could split the sentence into two sentences.

Reply:
Hi @chikoski, feel free to submit PRs to improve the original sentences! Many of them were written by me, who is not a native English speaker 😄

Reply:
Thanks! 

As for the English sentence, it looks good to me, actually. I can found may English sentences that connect two clauses mentioning different things with "and". It is OK for me. 

The original translation proposal looks strange because of the way of connecting two clauses. We, at least me, expect a parallel structure between two clauses when the two clauses have different subject and connected with a comma (、), such as "Futures are polled by an async runtime, and MyFuture is not".

Reply:
Okay, good to hear! I just wanted to mention it: as translators, you have a unique view of the course since you're working your way through the entire thing line-by-line. Most people only skim the text a bit here and there :smile:

Reply:
Hi, thank you @chikoski and @mgeisler for feedback. Basically plus one to @chikoski and I'll reflect to the next commit.